common/src/main/java/tachyon/security/CustomLoginModule.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security;
+
+import java.util.Map;
+
+import javax.security.auth.Subject;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.login.LoginException;
+import javax.security.auth.spi.LoginModule;
+
+import tachyon.Constants;
+
+/**
+ * A custom login module that creates a user based on the user name provided through application
+ * configuration.
+ */
+public class CustomLoginModule implements LoginModule {
+  private Subject mSubject;
+  private User mUser;
+
+  @Override
+  public void initialize(Subject subject, CallbackHandler callbackHandler,
+      Map<String, ?> sharedState, Map<String, ?> options) {
+    mSubject = subject;
+  }
+
+  /**
+   * Retrieve the user name by querying the property of Constants.TACHYON_SECURITY_USERNAME.
+   *
+   * @return true if customized user name is not null or empty
```

Comment:
"@return true if customized user name is not null or empty", this statement is misleading, as I thought it returns true if ... is empty, which is not the case.